concourse/setup.py

```diff
@@ -167,32 +168,76 @@ def create_instance_specific_helm_values(concourse_cfg: ConcourseConfig):
     concourse_tls_secret_name = concourse_cfg.tls_secret_name()
     ingress_host = concourse_cfg.ingress_host()
 
-    instance_specific_values = {
-        'concourse': {
-            'externalURL': external_url,
-            'githubAuth': {
-                'team': creds.github_auth_team(),
-                'authUrl': creds.github_auth_auth_url(),
-                'tokenUrl': creds.github_auth_token_url(),
-                'apiUrl': creds.github_auth_api_url(),
+    concourse_version = concourse_cfg.concourse_version
+    if concourse_version == "3":
```

Comment:
should the differentiation not rather be done based on the helm-chart version? OTOH, this will obviously work for now; at least a comment might make sense as this could be hard to understand "next time"

Reply:
Added a comment